.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,67 +1,72 @@
-[Short description explaining the high-level reason for the pull request]
+_[Enter an explanation of what the pull request does and why.]_
 
-## Additions
 
--
-
-## Removals
-
--
+---
 
-## Changes
+_[Feel free to delete any checkboxes that are not applicable to this PR.]_
 
--
 
-## Testing
+## How to test this PR
 
 1.
 
+
 ## Screenshots
 
 
-## Notes
+## Notes and todos
 
 -
 
-## Todos
-
--
 
 ## Checklist
 
+_[Feel free to delete any checkboxes that are not applicable to this PR.]_
+
 - [ ] PR has an informative and human-readable title
+  - PR titles are used to generate the change log in [releases](../../releases),
+    so good ones make that easier to scan.
+  - Consider prefixing with the feature/section you're addressing,
+    e.g., "Mega Menu: fix layout bug" or "Paying for College: Update content on Repay tool"
 - [ ] Changes are limited to a single goal (no scope creep)
-- [ ] Code can be automatically merged (no conflicts)
 - [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
```

Comment:
I'd love to see this one expanded into multiple actionable checks, but that can happen in a future PR.

Reply:
Yes, I still would like to, as well, but it seemed a bit too large of a subtask for me to take on myself.